locales/en/bento.ftl

```diff
@@ -0,0 +1,10 @@
+bento-button-title = Firefox apps and services
+fx-makes-tech = Firefox is tech that fights for your online privacy.
+made-by-Mozilla = Made by Mozilla
+fx-desktop = Firefox Browser for Desktop
+fx-mobile = Firefox Browser for Mobile
+fx-lockwise = Firefox Lockwise
```

Comment:
I think we should create terms for all these brands in app.ftl (Lockwise, Pocket, Monitor, Send), and here reference those terms.